### PR TITLE
Self cal name consistency

### LINKED ISSUE #615: Self calibration function name is inconsistent across drivers
```
us-aus-ras-130-164-29-8:nimi-python kirsch$ find generated | grep session\.py | xargs grep -irh "def\ .*self_cal" | sort
    def cal_self_calibrate(self, option):
    def get_self_cal_last_date_and_time(self):
    def get_self_cal_last_date_and_time(self):
    def get_self_cal_last_temp(self):
    def get_self_cal_last_temp(self):
    def get_self_cal_supported(self):
    def get_self_cal_supported(self):
    def self_cal(self):
    def self_cal(self):
```

This can be annoying to the Python customer because I can see iterating over a list of sessions calling things like self_cal. This would break it.

It's not clear whether we want to do it or not, since we are keeping to the C interfaces and inheriting those faults. This falls a little outside of the scope of what we signed up for.

### LINKED ISSUE #787: There is no way to self cal a dcpower device
nidcpower is missing self_cal

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- [X] I've added/updated tests applicable for this pull request
### What does this Pull Request accomplish?
* Enables self cal for DCPower
* Make self cal function names consistent accross drivers - `self_cal()`

### List issues fixed by this Pull Request below, if any.
* Fix #615 
* Fix #787

### What testing has been done?
* Unit
* System

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -80,13 +80,14 @@ All notable changes to this project will be documented in this file.
             * `VideoSignalFormat._1080I_59_94_FIELDS_PER_SECOND` --> `VideoSignalFormat.VIDEO_1080I_59_94_FIELDS_PER_SECOND`
             * `VideoSignalFormat._1080I_60_FIELDS_PER_SECOND` --> `VideoSignalFormat.VIDEO_1080I_60_FIELDS_PER_SECOND`
             * `VideoSignalFormat._1080P_24_FRAMES_PER_SECOND` --> `VideoSignalFormat.VIDEO_1080P_24_FRAMES_PER_SECOND`
-        * Removed `number_of_coefficients` parameter from `get_equalization_filter_coefficients()`
+        * Self cal function renamed to `self_cal()` to match other drivers - issue [#615](https://github.com/ni/nimi-python/issues/615)
```
Comment (marcoskirsch) [inline CHANGELOG.md]: `niscope.Session.cal_self_calibrate` renamed to ...

More precise.

File: docs/nidcpower/session.rst [hunk 2]
```diff
@@ -360,6 +360,8 @@ nidcpower.Session
    +------------------------------------------------------------+
    | :py:func:`reset_with_defaults`                             |
    +------------------------------------------------------------+
+   | :py:func:`self_cal`                                        |
```
Comment (marcoskirsch) [inline docs/nidcpower/session.rst]: this addition is not noted in CHANGELOG.md

Comment (texasaggie97) [reply docs/nidcpower/session.rst]: Added

File: CHANGELOG.md (hunk 1 above)
Comment (texasaggie97) [reply CHANGELOG.md]: Changed

**Pull request merged**